docs/basics/paintbrush.md

```diff
@@ -37,3 +37,11 @@ Clicking the **Use 3D brushstrokes** button will activate the 3D brush, 3D annot
 The transparency of active and non-active annotations can be altered by clicking the **Annotation Transparency** button.
 Using the two sliders allows you to modify the transparency of both the current (brush) and all other (brush) annotations in the range of 0-100% (opacity).
 This can be a very useful tool to determine the accuracy of drawn annotations by comparing to the background image.
+
+## Strokes vs. Pixels
+
+Brush strokes can be converted from the standard vector-graphic representation into pixel level annotations, based on the size of the pixels of the image being annotated.
+
+![Converter Button](/img/annotate/annotate_strokes.png)
```

Comment:
image appears broken in preview

Reply:
working in built preview: https://gray-island-021371003-38.westeurope.1.azurestaticapps.net/docs/basics/paintbrush